File: 2023/day5/part2.js
const fs = require("fs")

let lines = fs.readFileSync("input.csv", {encoding: "utf-8"}).split("\n")

for (let i=0; i<lines.length; i++) {
    lines[i] = lines[i].slice(0,-1)
}

function main(lines) {
    console.log(lines)
    
}

masterNumber = main(lines)

File: 2025/template/part2.py
def read_file(filename):
    with open(filename, encoding="utf-8") as file:
        lines = [line.split() for line in file.read().split("\n")]
        return lines
    
def main():
    answer = 0
    filename = "input.csv"
    lines = read_file(filename)

    return answer

answer = main()
print(answer)
